File: taller/AMarotoChivite-taller/AMarotoChivite-taller-Java/src/test/java/simulacionTaller/factories/MultiFactoryTest.java
package simulacionTaller.factories;

import org.junit.jupiter.api.Test;
import simulacionTaller.models.Multi;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class MultiFactoryTest {

    @Test
    void createTest() {
        Multi multi = MultiFactory.create();
        // No nulo, para ver si se han asignado correctamente los datos
        assertNotNull(multi);
    }
}

File: taller/AMarotoChivite-taller/AMarotoChivite-taller-kotlin/src/test/kotlin/simulacionTaller/factories/MultiFactoryTest.kt
package simulacionTaller.factories

import org.junit.jupiter.api.Assertions
import org.junit.jupiter.api.Test

internal class MultiFactoryTest {
    @Test
    fun createTest() {
        val multi = MultiFactory.create()
        // No nulo, para ver si se han asignado correctamente los datos
        Assertions.assertNotNull(multi)
    }
}